projects/plugins/jetpack/extensions/blocks/cookie-consent/index.js

```diff
@@ -0,0 +1,61 @@
+import { __ } from '@wordpress/i18n';
+import { getIconColor } from '../../shared/block-icons';
+import attributes from './attributes';
+import edit from './edit';
+import icon from './icon';
+import save from './save';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'cookie-consent';
+export const title = __( 'Cookie Consent', 'jetpack' );
+export const cookieName = 'eucookielaw';
+export const settings = {
+	title,
+	description: __(
+		'To display this block on all pages of your site, please add it inside a Template Part that is present on all your templates, like a Header or a Footer.',
```

Comment:
Maybe there could be a short sentence describing what this is for, and then this sentence explaining how to use it?